ipc/tests/vmPart_test.go

```diff
@@ -83,7 +83,7 @@ func doContractRequest(
 			},
 		}
 
-		part, err := arwenpart.NewArwenPart(
+		part, err := vmpart.NewArwenPart(
```

Comment:
rename also the constructor? (Arwen)

Reply:
Will definitely do in the next PR. This PR is only concerned with files / folders / packages (structural changes) and minimally editing the content so that tests pass.

The next PR modifies the actual code (content changes) in detail.